Added playbackRate stuff to duration stuff in FunkinLua

rodney528 opened the pull request.

rodney528 committed:
Added isStagePixel and curStage for if statements

rodney528 committed:
Update FunkinLua.hx

rodney528 committed:
Update FunkinLua.hx

rodney528 committed:
Added some stuff to FunkinLua.hx

rodney528 committed:
Opps

I made it the health bar not the health amount.

rodney528 committed:
Added chartingMode

Name says it all but ex:
if chartingMode then
    -- Any thing here will only appear in charting mode!
end

rodney528 committed:
God dang it

getHealth already existed

rodney528 committed:
Update... yeah.

I think I needed to add instance? I really need to figure out how to compile to test this shit.

rodney528 committed:
Equal amount of binds

keyJustPressed, keyPressed and keyReleased now all have the ability to grab the same buttons!

rodney528 committed:
Merge branch 'main' of https://github.com/ImaginationSuperHero528/FNF-PsychEngine

rodney528 committed:
Added setOnLuas
Like the title says I added setOnLuas... cause why not?

rodney528 committed:
Forget the ; opps

rodney528 committed:
Small Change

Should probably be set as `Dynamic` and not `String`.

rodney528 committed:
Easy use of getProperty

Only trying to do ones that need getPropertyFromClass (I think that's one I wanna refer to?)

rodney528 committed:
I'm basically removing anything using getProperty

rodney528 committed:
If "dad.curCharacter" is same as "gf.curCharacter"

Basically ya know how in the tutorial gf is technically the dad? Well this makes it (I hope) so it can work with anyone other then just gf.

rodney528 committed:
playbackRate to duration stuff in FunkinLua

rodney528 committed:
did this too ig

rodney528 committed:
Did more of PlayState

CaptainBaldi commented:
this should've been made to begin with, noticed this issue while messing with some of my ports on the later actuons

rodney528 committed:
Made it if equals now

dad.curCharacter.startsWith(gf.curCharacter) has been replaced with dad.curCharacter == gf.curCharacter

rodney528 committed:
Still working on :ShadowTroll:

rodney528 committed:
:P

rodney528 committed:
For duration it should be / woops!

rodney528 committed:
Screw it, fuck if easy use

rodney528 committed:
Merge branch 'main' into playbackRate-to-duration-stuff-in-FunkinLua

rodney528 committed:
progress ig

rodney528 committed:
Changed up

rodney528 committed:
Uh... fixed?

rodney528 committed:
Revert "Uh... fixed?"

This reverts commit f6973c0b984068100144503994bd0a99044585d1.

rodney528 committed:
Revert "Changed up"

This reverts commit 8811146d59a06555bc5e36c6e00dc1f8a387fc05.

rodney528 committed:
Hopefully it's fixed now

rodney528 committed:
it's been a bit

thx kingyomoma

Co-Authored-By: kingyomoma <96543290+kingyomoma@users.noreply.github.com>

rodney528 committed:
:whyyyyy:

rodney528 committed:
Merge branch 'setOnLuas'

rodney528 committed:
Merge branch 'loadSong-FIX'

rodney528 committed:
Merge branch 'playbackRate-to-duration-stuff-in-FunkinLua'

rodney528 committed:
Merge branch 'if-dad-is-same-as-gf'

rodney528 committed:
Merge branch 'Lua-Adds'

rodney528 committed:
Revert "Merge branch 'Lua-Adds'"

This reverts commit b7f13616fe8fd5df604094eaeb9b62d31549d322, reversing
changes made to b4e8964711874b4f0ccc5eb9e1a9f37ad60762d3.

rodney528 committed:
Revert "Merge branch 'if-dad-is-same-as-gf'"

This reverts commit b4e8964711874b4f0ccc5eb9e1a9f37ad60762d3, reversing
changes made to 46c5d94eb1274b45cf9e3daa883a388bf8a8af74.

rodney528 committed:
Revert "Merge branch 'playbackRate-to-duration-stuff-in-FunkinLua'"

This reverts commit 46c5d94eb1274b45cf9e3daa883a388bf8a8af74, reversing
changes made to 7606d1597f246df37a613075a7c6d538916abf0a.

rodney528 committed:
Revert "Merge branch 'loadSong-FIX'"

This reverts commit 7606d1597f246df37a613075a7c6d538916abf0a, reversing
changes made to 2ad13aaa39ad9f4046263265097a1b4ec6f61dc5.

rodney528 committed:
Revert "Merge branch 'setOnLuas'"

This reverts commit 2ad13aaa39ad9f4046263265097a1b4ec6f61dc5, reversing
changes made to bdb6910951cc1a28c8c6a8898091789287f70004.
